extensions/keccak256/guest/src/lib.rs

```diff
@@ -40,7 +20,7 @@ pub fn keccak256(input: &[u8]) -> [u8; 32] {
 #[cfg(target_os = "zkvm")]
 #[inline(always)]
 #[no_mangle]
-extern "C" fn native_keccak256(bytes: *const u8, len: usize, output: *mut u8) {
+pub extern "C" fn native_keccak256(bytes: *const u8, len: usize, output: *mut u8) {
```

Comment:
is the `pub` needed? I just want to make sure it works with a non-`pub` extern: https://docs.rs/alloy-primitives/latest/src/alloy_primitives/utils/mod.rs.html#166